allennlp/service/predictors/simple_seq2seq.py

```diff
@@ -0,0 +1,19 @@
+from typing import Tuple
+from overrides import overrides
+
+from allennlp.common.util import JsonDict
+from allennlp.data import Instance
+from allennlp.service.predictors.predictor import Predictor
+
+@Predictor.register('simple_seq2seq')
+class SimpleSeq2SeqPredictor(Predictor):
+    """
+    Wrapper for the :class:`~allennlp.models.encoder_decoder.simple_seq2seq` model.
+    """
+    @overrides
+    def _json_to_instance(self, json_dict: JsonDict) -> Tuple[Instance, JsonDict]:
+        """
+        Expects JSON that looks like ``{"source_string": "..."}``.
```

Comment:
Maybe no need to describe the type here, I think - just "source" or "source_sentence" would be fine.

Reply:
Hello, why naming "source_string" is just due to the consideration of being consistent to the argument name at seq2seq dataset reader's text_to_instance() 
See line at
https://github.com/allenai/allennlp/blob/master/allennlp/data/dataset_readers/seq2seq.py#L81

Reply:
@DeNeutoy, yeah, I agree, I think "source" is better here.  But I'm going to change that in a subsequent PR, because something got messed up with the git repo this PR is coming from (it says from "unknown repository" above).